packages/jest-circus/src/testCaseReportHandler.ts

```diff
@@ -0,0 +1,36 @@
+/**
+ * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import type {Circus} from '@jest/types';
+import type {TestCase, TestCaseResult} from '@jest/test-result';
+import {makeSingleTestResult, parseSingleTestResult} from './utils';
+
+const testCaseReportHandler = (
+  testPath: string,
+  sendMessageToJest: Function,
+) => (event: Circus.Event) => {
+  switch (event.name) {
+    case 'test_done': {
+      const testResult = makeSingleTestResult(event.test);
+      const testCaseResult: TestCaseResult = parseSingleTestResult(testResult);
```

Comment:
```suggestion
      const testCaseResult = parseSingleTestResult(testResult);
```

`parseSingleTestResult` should have the return type, so this isn't needed